front/src/components/messaging/MessageChannelItem.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC } from 'react';
+import { Image, StyleSheet, View } from 'react-native';
+import { Text, TouchableRipple } from 'react-native-paper';
+
+import { MessageChannel } from '@/models/entities/messageChannel';
+
+/**
+ * The styles for the MessageChannelItem component.
+ */
+const styles = StyleSheet.create({
+  container: {
+    gap: 16,
+  },
+  horizontalContainer: {
+    flexDirection: 'row',
+    gap: 12,
+  },
+  nameText: {
+    marginBottom: 5,
+  },
+  profilePicture: {
+    borderRadius: 48,
+    height: 60,
+    width: 60,
+  },
+});
+
+/**
+ * The props for the MessageChannelItem component.
+ */
+type MessageChannelItemProps = {
+  /**
+   * The function to call when an a message channel is pressed.
+   */
+  onItemPress?: (messageChannel: MessageChannel) => void;
+
+  /**
+   * The message channel to display.
+   */
+  messageChannel: MessageChannel;
+};
+
+/**
+ * Displays a message channel.
+ * @constructor
+ */
+const MessageChannelItem: FC<MessageChannelItemProps> = ({
+  onItemPress,
+  messageChannel,
+}) => {
+  return (
+    <TouchableRipple onPress={() => onItemPress?.(messageChannel)}>
+      <View style={[styles.container, styles.horizontalContainer]}>
+        <Image
+          style={styles.profilePicture}
+          source={{
+            uri: messageChannel.employer?.picture,
+          }}
+        />
+        <View>
+          <Text
+            variant='labelLarge'
+            style={styles.nameText}
+          >{`${messageChannel.employer.firstName} ${messageChannel.employer.lastName}`}</Text>
+
+          <Text>{`${messageChannel.lastMessage}`}</Text>
+        </View>
```

Comment:
Can you vertically center those two texts?

Reply:
![image](https://github.com/thomas-mauran/LinkedOut/assets/78204354/1dbd47b7-feac-482e-b1f9-b2f1ce7ead41)
It seems a bit odd centered no ?
tell me what you think

Reply:
ping @Kuruyia

Reply:
Damn! I meant in the other axis, so both texts are centered with the profile picture.

Reply:
![image](https://github.com/thomas-mauran/LinkedOut/assets/78204354/9f50b66f-e575-4164-94da-ecdf22cdd791)


done here it what it looks like

Reply:
I think it looks better!